src/main/deploy/pathplanner/settings.json

```diff
@@ -0,0 +1,39 @@
+{
+  "robotWidth": 0.743,
+  "robotLength": 0.8509,
+  "holonomicMode": true,
+  "pathFolders": [
+    "New Folder"
```

Comment:
Can we have more descriptive names other than "New Folder" and "New New Folder"?